Gopkg.toml

```diff
@@ -104,3 +104,7 @@ required = [
 [[constraint]]
   name = "github.com/open-policy-agent/opa"
   version = "0.5.13"
+
+[[constraint]]
+  name = "github.com/onsi/gomega"
+  revision = "ba3724c94e4dd5d5690d37c190f1c54b2c1b4e64"
```

Comment:
Can we pin to a release rather than a SHA here?

Reply:
Google security practice is to use crypto hash instead of modifiable git tags.

Reply:
I thought I read in our docs that we were going to prefer releases in the toml and only pin to a SHA as an exception. But I can't seem to dig up where I read that. Today we have a mix. /cc @costinm

Reply:
It has happened in the past that some projects simply remove/re-create git tag for a version, which can affect our build in a negative way.

Reply:
Right. I don't particularly care either way, I only commented because I read that guidance about the new build system early today/yesterday.